skietbaan_fe/src/components/Login.js

```diff
@@ -1,15 +1,15 @@
 import React, { Component } from 'react';
-import { 
-  Col, 
-  FormGroup, 
-  Button, 
-  Form 
+import {
+  Col,
+  FormGroup,
+  Button,
+  Form
 } from 'reactstrap';
 import '../components/RegisterStyles.css';
-import {  validateUsername } from './Validators.js';
+import { validateUsername } from './Validators.js';
 import { getCookie } from './cookie.js';
-import {URL} from '../actions/types.js';
-import header from'../components/assets/header.png';
+import { BASE_URL, URL } from '../actions/types.js';
```

Comment:
Each on its own line please 
```
import {
  BASE_URL,
  URL
}
```

Reply:
Im not using Base url I'll just remove it